src/main/java/org/skriptlang/skriptworldguard/elements/expressions/ExprRegionFlag.java

```diff
@@ -0,0 +1,131 @@
+package org.skriptlang.skriptworldguard.elements.expressions;
+
+
+
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+
+import com.sk89q.worldguard.WorldGuard;
+import com.sk89q.worldguard.protection.flags.*;
+
+import com.sk89q.worldguard.protection.regions.ProtectedRegion;
+
+
+import org.bukkit.event.Event;
+import org.skriptlang.skriptworldguard.SkriptWorldGuard;
+import org.skriptlang.skriptworldguard.worldguard.WorldGuardRegion;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class ExprRegionFlag extends PropertyExpression<WorldGuardRegion, String> {
+
+    static {
+
+        register(ExprRegionFlag.class, String.class, "[worldguard] flag %string%", "worldguardregions");
+

```

Comment:
Would changing this expression to "combined" (from property) require to change the class of the expression from PropertyExpressiom to SimpleExpression?

Reply:
I'm not sure if it is necessary to change it from Property - it can be a little odd sometimes to include an additional expression in a property expression but in this case it should work

Reply:
I mean it does and can work indeed, it's just not the correct thing to do 😬